From: Vagrant Cascadian <vagrant@reproducible-builds.org>
Date: Sun, 30 Apr 2023 15:44:21 -0700
Subject: man/fill_template: Use numeric year-month-date for manpage.

The month may be rendered for the locale of the build environment.

https://reproducible-builds.org/docs/locales/

Forwarded: https://github.com/jackaudio/jack2/pull/932
---
 man/fill_template | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/man/fill_template b/man/fill_template
index 368cb1b..d1df18d 100644
--- a/man/fill_template
+++ b/man/fill_template
@@ -4,8 +4,8 @@ d=""
 
 if [ "$2" == "True" ]; then
   for i in *.0 ; do
-    sed -e "s/!VERSION!/${1}/g" -e "s/!DATE!/`date $d '+%B %Y'`/g" < ${i} > ${i%%0}1
+    sed -e "s/!VERSION!/${1}/g" -e "s/!DATE!/`date $d '+%Y-%m-%d'`/g" < ${i} > ${i%%0}1
   done
 else
-  sed -e "s/!VERSION!/${1}/g" -e "s/!DATE!/`date $d '+%B %Y'`/g" < jackd.0 > jackd.1
+  sed -e "s/!VERSION!/${1}/g" -e "s/!DATE!/`date $d '+%Y-%m-%d'`/g" < jackd.0 > jackd.1
 fi
